.github/workflows/docs.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           fetch-depth: 1
       - name: Get subprojects that have doc changes
         id: docs-changed-subprojects
-        uses: tj-actions/changed-files@fea790cb660e33aef4bdf07304e28fedd77dfa13 # v39.2.4
+        uses: tj-actions/changed-files@dcc7a0cba800f454d79fff4b993e8c3555bcc0a8 # v45.0.7
```

Comment:
For reference, breaking changes since v39.2.4:
- https://github.com/tj-actions/changed-files/releases/tag/v40.0.0
- https://github.com/tj-actions/changed-files/releases/tag/v41.0.0
- https://github.com/tj-actions/changed-files/releases/tag/v42.0.0
- https://github.com/tj-actions/changed-files/releases/tag/v43.0.0
- https://github.com/tj-actions/changed-files/releases/tag/v44.0.0
- https://github.com/tj-actions/changed-files/releases/tag/v45.0.0

Doesn't seem like any action is required, but leaving it here just in case.